### PR TITLE
Add snapshot e2e testcases

### PR DESCRIPTION
**Pull request by meghnasiingh**
Signed-off-by: Meghna Singh <meghna@Meghnas-MacBook-Pro.local>

### CODE REVIEW

File: e2e-final/config/test-cases-block.yml [hunk 1]
```diff
@@ -163,4 +163,20 @@
       iops: 2000
       tags: "Testing VPC volume from library with invalid custom iops"
     vpcZone: eu-gb-2
-  success: false
\ No newline at end of file
+  success: false
```
Comment (sameshai) [inline e2e-final/config/test-cases-block.yml:166]: As of now this looks good can we see if we can add negative test cases here ?

**Review by sameshai — APPROVED** [review]
LGTM , please open GHE to see if it is feasible to add negative cases.

File: e2e-final/config/test-cases-block.yml (hunk 1 above)
Comment (ambiknai) [reply e2e-final/config/test-cases-block.yml:166]: https://github.ibm.com/alchemy-containers/armada-storage/issues/3831

File: e2e-final/scripts/create_ins-bak.sh [hunk 2]
```diff
@@ -0,0 +1,77 @@
+#!/bin/sh
```
Comment (prankulmahajan) [inline e2e-final/scripts/create_ins-bak.sh:1]: Is the file name `create_ins-bak.sh` intended?

Comment (ambiknai) [reply e2e-final/scripts/create_ins-bak.sh]: Renamed

**Review by prankulmahajan — APPROVED** [review]
lgtm

**Pull request merged**